docker/Dockerfile-workers

```diff
@@ -1,13 +1,15 @@
 # syntax=docker/dockerfile:1
 
 ARG SYNAPSE_VERSION=latest
+# This one doesn't get hardcoded, as it's an intermediate build step for Complement and
+# is not pulled from a registry.
```

Comment:
```suggestion
```

Thinking about it, I would probably just remove this comment — this Dockerfile is used for not just Complement and in some cases it *is* pulled from the registry. But I'm happy to leave this one ambiguous/not fully qualified.
The comment just feels a bit out of place and only seems to make sense if you're looking at it from this PR. It's possibly easier to just leave it as before.